src/geo/gmaps/gmaps-cartodb-layer-group-view.js

```diff
@@ -235,7 +235,7 @@ _.extend(
     _reload: function () {
       var tileURLTemplates;
       if (this.model.hasTileURLTemplates()) {
-        tileURLTemplates = this.model.getTileURLTemplates();
+        tileURLTemplates = [this.model.getTileURLTemplate()];
```

Comment:
We could use something like what I suggested in https://github.com/CartoDB/cartodb.js/pull/1602/files#r107707428 here.